activerecord/lib/active_record/persistence.rb

```diff
@@ -621,8 +621,12 @@ def reload(options = nil)
 
     # Saves the record with the updated_at/on attributes set to the current time
     # or the time specified.
-    # Please note that no validation is performed and only the +after_touch+,
-    # +after_commit+ and +after_rollback+ callbacks are executed.
+    #
+    # - No validation is performed.
```

Comment:
Looking at other lists in the docs, it appears `*` should be used for delimiting lists. Can you change your `-` to `*`s? 😄